### PR TITLE
LG-2499 test module builds

### PR DESCRIPTION
**Pull request by TheSonOfThomp**
## ✍️ Proposed changes

Adds a check to ensure every changed package builds correctly

🎟 _Jira ticket:_ [LG-2499](https://jira.mongodb.org/browse/LG-2499)

## 🛠 Types of changes

- [x] Tooling (updates to workspace config or internal tooling)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Style change (purely visual updates; if this is the case, attach a screenshot below!)

## ✅ Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have run `yarn changeset` and documented my changes

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README

## Feedback Levels

Please try to mark your comments with one of these levels

1. Praise (👏 🚀 🔥 🍾)
2. Opinions, Ideas, Suggestions (💡 👀)
3. Let’s discuss @ Code review (🤷 🤔 🧐 )
4. Doesn’t meet agreed upon standards (😬 ⚠️ ‼️)
5. Missing an edge case (🛑 🙅 ❌)
6. Missing core functionality (⛔️ ☣️ 🚧 🌋 💀)

(Note: Use ⌘-⌃-␣ to open the Emoji keyboard!)

### CODE REVIEW

File: scripts/test.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ interface Opts {
 }
 
 const cli = new Command('test')
-  .description('Tests leagygreen-ui packages.')
```
Comment (spark33) [inline scripts/test.ts:22]: 😂

File: package.json [hunk 2]
```diff
@@ -26,20 +26,21 @@
...
     "check": "npm-run-all --parallel check:*",
     "check:prettier": "prettier --check \"**/*.{js,ts,tsx,json,md}\"",
     "check:eslint": "eslint \"**/*.{js,ts,tsx}\"",
     "check:deps": "ts-node -O '{\"module\": \"commonjs\", \"resolveJsonModule\": true}' ./scripts/depcheck.ts",
+    "check:builds": "yarn ts-node ./scripts/check-builds.ts",
     "check:ts": "tsc --project tsconfig.json",
     "clean": "yarn clean:builds && yarn clean:modules",
-    "clean:builds": "rm -rf packages/*/dist packages/*/tsconfig.tsbuildinfo",
-    "clean:modules": "rm -rf packages/*/node_modules node_modules",
+    "clean:builds": "rm -rf packages/*/dist packages/*/tsconfig.tsbuildinfo website/tsconfig.tsbuildinfo ./tsconfig.tsbuildinfo",
```
Comment (spark33) [inline package.json:39]: I assume this website/... will go away pretty soon?

**Review by spark33 — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (TheSonOfThomp) [reply package.json:39]: yeah 👍

File: scripts/test.ts (hunk 1 above)
Comment (TheSonOfThomp) [reply scripts/test.ts:22]: 🫣
